$NetBSD: patch-configure,v 1.1 2013/12/12 15:33:21 noud4 Exp $

Get the man dir right.
Get the GeoIP.dat path right.

--- configure.orig	2013-11-19 22:55:00.000000000 +0000
+++ configure
@@ -352,6 +352,8 @@ public:
                 }
                 #ifdef Q_OS_FREEBSD
                   conf->addLib("-lexecinfo");
+                #endif
+                #if defined(Q_OS_FREEBSD) || defined(Q_OS_NETBSD)
                   conf->addExtra("MANPREFIX = \$\$PREFIX");
                 #else
                   conf->addExtra("MANPREFIX = \$\$PREFIX/share"); 
@@ -562,7 +564,7 @@ public:
 			return true;
 #ifdef Q_WS_X11
 		}
-                if(QFile::exists("/usr/share/GeoIP/GeoIP.dat") || QFile::exists("/usr/local/share/GeoIP/GeoIP.dat") || QFile::exists("/var/lib/GeoIP/GeoIP.dat"))
+                if(QFile::exists("@PREFIX@/share/GeoIP/GeoIP.dat") || QFile::exists("/usr/local/share/GeoIP/GeoIP.dat") || QFile::exists("/var/lib/GeoIP/GeoIP.dat"))
 			return true;
                 printf("\nWarning: GeoIP database was not found at /usr/share/GeoIP/GeoIP.dat or /var/lib/GeoIP/GeoIP.dat\nCountry resolution will be slow.");
 		return false;
